src/Core/ViewHelper/Traits/ParserRuntimeOnly.php

```diff
@@ -29,6 +29,6 @@ public function render()
      */
```

Comment:
Should we not add at least the return type as docblock here ?

```
@return string
```

Reply:
done